tidb-configuration-file: add detail for graceful shutdown (#21980)
pingcap/docs: tidb-configuration-file.md

- Specifies the number of seconds that TiDB waits when you shut down the server, which allows the clients to disconnect.
- Default value: `0`

> - When TiDB is waiting for shutdown (in the grace period), the HTTP status will indicate a failure, which allows the load balancers to reroute traffic. It also responds with an error to the `COM_PING` command.

Reviewer: ```suggestion
- When TiDB is waiting for shutdown (in the grace period), the HTTP status will indicate a failure, which allows the load balancers to reroute traffic. TiDB also returns an error in response to the `COM_PING` command.
```
